src/quo2/components/bottom_sheet/view.cljs

```diff
@@ -99,7 +100,7 @@
                                      (reset! show-bottom-sheet? false)
                                      (when (fn? on-cancel) (on-cancel))
                                      (timer/set-timeout
-                                      #(do (re-frame/dispatch [:bottom-sheet/hide]))
+                                      #(do (>evt [:bottom-sheet/hide-navigation-overlay]))
```

Comment:
we don't use evt anymore, use rf/dispatch instead

Reply:
oh ok, didn't know that